src/frontend/efiling-frontend/package.json

```diff
@@ -27,7 +27,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.3",
     "regenerator-runtime": "^0.13.7",
-    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/1.2.0/shared-components-1.2.0.tgz"
+    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/v1.2.1/shared-components-1.2.1.tgz"
```

Comment:
Out of curiosity, why is the new path /v1.2.1/ instead of .../1.2.1/... ?

Reply:
ah that would be a labeling mistake on my part when creating the release.